frontend/src/components/Activity.svelte

```diff
@@ -1,16 +1,5 @@
 <script lang="ts">
-    /* eslint-disable import/first */
-    import type { Status } from "$types/status";
-
-    interface Activity {
-        time: string;
-        symbol: string;
-        side: string;
-        qty: number;
-        price: number;
-        totalVal: number;
-        status: Status;
-    }
+    import type { Activity } from "$lib/types";
```

Comment:
Eventually all types will be specified in lib (so we can check them directly against response schemas), so this is pre-emptive of that.